assets/route53.yaml

```
@@ -0,0 +1,19 @@
+---
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: gitpod-issuer
+spec:
+  acme:
+    email: $LETSENCRYPT_EMAIL
+    privateKeySecretRef:
+      name: issuer-account-key
+    server: https://acme-v02.api.letsencrypt.org/directory
+    solvers:
+      - dns01:
+          route53:
+            region: us-west-1
```

It feels like the region should be configurable. Remember to add the variable to `.env.example`

```suggestion
            region: ${ROUTE53_REGION}
```

---

Will do.